clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py

```diff
@@ -14,60 +14,75 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from collections import namedtuple
 from grafanalib import core as g
 import defaults as d
 
 
-def api_call_latency(title, verb, scope, threshold):
-    return d.Graph(
-        title=title,
-        targets=[
-            g.Target(expr=str(threshold), legendFormat="threshold"),
-            g.Target(
-                expr=d.one_line(
-                    """
+
+
+def api_call_latency_panel(expression):
+    def api_call_latency(title, verb, scope, threshold, expression):
+        return d.Graph(
+            title=title,
+            targets=[
+                g.Target(expr=str(threshold), legendFormat="threshold"),
+                g.Target(
+                    expr=d.one_line(expression % {"verb": verb, "scope": scope}
+                                    ),
+                    # TODO(github.com/grafana/grafana/issues/19410): uncomment once fixed
+                    # legendFormat="{{verb}} {{scope}}/{{resource}}",
+                ),
+            ],
+            yAxes=g.single_y_axis(format=g.SECONDS_FORMAT),
+        )
+
+    ApiCallLatencyPanelArgs = namedtuple('ApiCallLatencyPanelArgs', ['title', 'verb', 'scope', 'threshold'])
+
+    API_CALL_LATENCY_PANELS_TEMPLATE = [
+        ApiCallLatencyPanelArgs(
+            title="Read-only API call latency (percentaile=99, scope=resource, threshold=1s)",
+            verb="GET",
+            scope="namespace",
+            threshold=1
+        ),
+        ApiCallLatencyPanelArgs(
+            title="Read-only API call latency (percentaile=99, scope=namespace, threshold=5s)",
+            verb="LIST",
+            scope="namespace",
+            threshold=5,
+        ),
+        ApiCallLatencyPanelArgs(
+            title="Read-only API call latency (percentaile=99, scope=cluster, threshold=30s)",
+            verb="LIST",
+            scope="cluster",
+            threshold=30,
+        ),
+        ApiCallLatencyPanelArgs(
+            title="Mutating API call latency (threshold=1s)",
+            verb=d.any_of("CREATE", "DELETE", "PATCH", "POST", "PUT"),
+            scope=d.any_of("namespace", "cluster"),
+            threshold=1,
+        ),
+    ]
+    return [api_call_latency(title=n.title, verb=n.verb, scope=n.scope, threshold=n.threshold, expression=expression) for n in API_CALL_LATENCY_PANELS_TEMPLATE]
```

Comment:
This should be written as `api_call_latency_panels_template`. All caps named variables are considered to be consts in Python. 

Since you will be changing this, I'd go for a shorter name (e.g. `args`, `panel_args`)

Reply:
Actually, it is const, so I do not plan to change it.

Why do you think it's not const?

Reply:
Const are module-level variables, that should never changed. The fact that it's not modified, doesn't make it a const - it's in-function variable, that simple doesn't change. 

Function variables should be typed as functions: using underscores. Please see this PEP: https://www.python.org/dev/peps/pep-0008/#prescriptive-naming-conventions.

Reply:
Refering to Pep: https://www.python.org/dev/peps/pep-0008/#constants
```
Constants are usually defined on a module level and written in all capital letters with underscores separating words.
```
They do no need to be defined on a module level, although I agree that this is the most common use case.

Could you point out what is the difference between all caps written variable on module level and on function level, and why one you are considering as a const and other as a regular mutable variable?

Reply:
Inlined variable so it does not need to have any name.

Reply:
O, that's even nicer! 

If you're still interested in discussion about naming convention, feel free to reach out to me offline.